services/filesstore/mocks/FileBackend.go

```diff
@@ -4,9 +4,12 @@
 
 package mocks
 
-import io "io"
-import mock "github.com/stretchr/testify/mock"
-import model "github.com/mattermost/mattermost-server/model"
+import (
+	io "io"
+
```

Comment:
Intended to be part of the change?

Reply:
Actually, wasn't intended, probably my auto go fmt execution did that.

Reply:
Fixed that